Directory.Build.props

```diff
@@ -2,9 +2,8 @@
   <ItemGroup>
     <PackageReference
       Include="SonarAnalyzer.CSharp"
-      Version="8.35.0.42613"
+      Version="8.46.0.54807"
       PrivateAssets="all"
-      Condition="$(MSBuildProjectExtension) == '.csproj'"
```

Comment:
We don't need that; the analyzer will automatically not run on a language it does not recognize.